app/services/invoices/payments/adyen_service.rb

```diff
@@ -30,8 +30,8 @@ def create
         res = create_adyen_payment
         return result unless res
 
-        handle_adyen_response(res)
-        return result unless result.success?
+        adyen_success, adyen_error = handle_adyen_response(res)
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `adyen_error`. Use `_` or `_adyen_error` as a variable name to indicate that it won't be used. (https://rubystyle.guide#underscore-unused-vars)